### PR TITLE
Bonsai gen_all_traj

### PR DESCRIPTION
**Pull request by GeorgeEfstathiadis**
Added gen_all_traj function to generate trajectories for a single person in a specified time-frame.

### CODE REVIEW

File: forest/bonsai/simulate_gps_data.py [hunk 1]
```diff
@@ -563,7 +547,7 @@ def calculate_trip(self, origin: Tuple[float, float],
             for try_no in range(3):
                 try:
                     path, _ = get_path(
-                        origin, destination, transport, api_key
+                        origin, destination, transport.value, api_key
```
Comment (hackdna) [inline forest/bonsai/simulate_gps_data.py:550]: Would it make more sense to update `get_path()` to use the `Vehicle` enum?

File: forest/bonsai/tests/test_simulate_gps_data.py [hunk 2]
```diff
@@ -1,12 +1,14 @@
 """Tests for simulate_gps_data module"""
 
+import datetime
+
```
Comment (hackdna) [inline forest/bonsai/tests/test_simulate_gps_data.py:4]: Extra blank line

**Review by hackdna — CHANGES_REQUESTED** [review]

File: forest/bonsai/tests/test_simulate_gps_data.py (hunk 2 above)
Comment (GeorgeEfstathiadis) [reply forest/bonsai/tests/test_simulate_gps_data.py:4]: Isn't this right? Since datetime is a standard library.

Comment (hackdna) [reply forest/bonsai/tests/test_simulate_gps_data.py:4]: Yes, indeed!

File: forest/bonsai/simulate_gps_data.py [hunk 3]
```diff
@@ -100,26 +108,26 @@ def get_path(start: Tuple[float, float], end: Tuple[float, float],
...
         path: 2d numpy array
-        transport: str
+        transport: Vehicle
     Returns:
         subset of original path that represents the flight
     """
 
     distance = great_circle_dist(
-        path[0][1], path[0][0], path[-1][1], path[-1][0]
+        *path[0], *path[-1]
```
Comment (hackdna) [inline forest/bonsai/simulate_gps_data.py:124]: This statement can now fit on one line.

**Review by hackdna — CHANGES_REQUESTED** [review]

**Review by hackdna — APPROVED** [review]
Thank you!

**Pull request merged**